packages/babel-preset/src/assets/replaceAssetsWithImports.ts

```diff
@@ -29,6 +30,39 @@ export function replaceAssetsWithImports(
     return assetIdentifiers.get(assetPath) as t.Identifier;
   }
 
+  function buildTemplateLiteralFromValue(value: string): t.TemplateLiteral {
+    const tokens = tokenize(value);
+
+    const quasis: t.TemplateElement[] = [];
+    const expressions: t.Identifier[] = [];
+
+    let acc = '';
+
+    for (let i = 0, l = tokens.length; i < l; i++) {
+      for (; i < l; i++) {
+        acc += tokens[i];
+
+        if (tokens[i] === 'url') {
+          const url = tokens[i + 1].slice(1, -1);
```

Comment:
why ignore the first and last characters of a url ?

Reply:
I'm going by your example of tokens: `('.foo', '{', 'url', 'image.png', '}')`

Reply:
The first and last characters are parens. `url(image.png)` tokenizes to `["url", "(image.png)"]`